packages/semantic-ui-theme/semantic/src/themes/sharegate/elements/label.overrides

```diff
@@ -283,3 +283,30 @@
 .ui.label.highlight {
     font-weight: 600;
 }
+
+/* CONTENT  | In Select */
```

Comment:
Not sure about this. I don't think a label should be aware of the fact that a multi-select component if consuming it?

I believe this code should be moved.